vundo.el

```diff
@@ -1150,6 +1187,13 @@ have a route from 3 to 2 (2’->3)."
                          return node
                          finally return nil)))
       (with-current-buffer buffer
+        ;; XXX TS-TRIM DEBUG CODE, REMOVE
+        (let ((bul buffer-undo-list)
+              (tul (vundo-m-undo-list possible-trim-point)))
+          (while (and bul (not (eq bul tul)))
+            (if (and (consp (car bul)) (eq (caar bul) t))
+                (message "Trimming TS at %s" (format-time-string "%F %r" (cdar bul))))
+            (setq bul (cdr bul))))
```

Comment:
This is where we could either save the master-equiv-of -> TS mapping in an alist in the orig-buffer, or (possibly), transport the TS itself to the undo directly after the master node.

Reply:
Cool. Don't forget to remove this code before merge tho.

Reply:
Done.